src/lib.rs

```diff
@@ -117,8 +120,17 @@ impl<const N: usize, A: Alphabet<N>> CustomSymbol<N, A> {
     /// representation. This is only possible because the [`u128`] used as the backing for
     /// [`CustomSymbol`] encodes all bits of information for each character in the
     /// [`CustomSymbol`].
-    pub fn to_string(&self) -> String {
-        self.into()
+    pub fn name(&self) -> String {
```

Comment:
So basically, I just renamed this `to_string` to `name` to avoid conflict with `ToString::to_string`.

I also moved conversion logic into this function because it made a bit more sense, but this is bikeshedding at this point haha.

Lmk if you prefer `to_name` to indicate work being done.

Reply:
Also, I'm not sure whether this qualifies as major semver change given that `ToString::to_string` still exists - but I'm assuming yes because using it requires `ToString` to be imported.

Reply:
yeah this should be a major version bump, but that's fine